File: lib/core/models/recruiter_info.dart
// Model for recruiterInfo table
class RecruiterInfo {
  final String idUser;
  final String? department;
  final String? title;
  final String? idCompany;

  RecruiterInfo({
    required this.idUser,
    this.department,
    this.title,
    this.idCompany,
  });

  Map<String, dynamic> toJson() {
    return {
      'idUser': idUser,
      'department': department,
      'title': title,
      'idCompany': idCompany,
    };
  }

  factory RecruiterInfo.fromJson(Map<String, dynamic> json) {
    return RecruiterInfo(
      idUser: json['idUser'] as String,
      department: json['department'] as String?,
      title: json['title'] as String?,
      idCompany: json['idCompany'] as String?,
    );
  }

  @override
  String toString() {
    return 'RecruiterInfo(idUser: $idUser, department: $department, title: $title, idCompany: $idCompany)';
  }
}

File: HuitWorks.WebAPI/Models/RecruiterInfo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HuitWorks.WebAPI.Models
{
    [Table("recruiterInfo")]
    public class RecruiterInfo
    {
        [Key]
        [Column("idUser")]
        [StringLength(64)]
        public required string IdUser { get; set; }

        [Column("title")]
        [StringLength(100)]
        public required string? Title { get; set; }

        [Column("idCompany")]
        [StringLength(64)]
        public string? IdCompany { get; set; }

        [Column("department")]
        [StringLength(100)]
        public string? Department { get; set; }

        [Column("description")]
        public string? Description { get; set; }
    }
}
